### PR TITLE
fixes #837

### LINKED ISSUE #837: SSO: Curl request doesn't work to get the token
This request fails 

```
curl -sk -X POST https://secure-sso-sso.ose.spring-boot.osepool.centralci.eng.rdu2.redhat.com/auth/realms/master/protocol/openid-connect/token \
-d grant_type=password -d username=admin -d password=admin \
-d client_secret=1daa57a2-b60e-468b-a3ac-25bd2dc2eadc -d client_id=demoapp

<html>
  <head>
    <meta name="viewport" content="width=device-width, initial-scale=1">

  <style type="text/css">
  /*!
   * Bootstrap v3.3.5 (http://getbootstrap.com)
   * Copyright 2011-2015 Twitter, Inc.
   * Licensed under MIT (https://github.com/twbs/bootstrap/blob/master/LICENSE)
   */
  /*! normalize.css v3.0.3 | MIT License | github.com/necolas/normalize.css */
  html {
    font-family: sans-serif;
    -ms-text-size-adjust: 100%;
    -webkit-text-size-adjust: 100%;
  }
  body {
    margin: 0;
  }
  h1 {
    font-size: 1.7em;
    font-weight: 400;
    line-height: 1.3;
    margin: 0.68em 0;
  }
  * {
    -webkit-box-sizing: border-box;
    -moz-box-sizing: border-box;
    box-sizing: border-box;
  }
  *:before,
  *:after {
    -webkit-box-sizing: border-box;
    -moz-box-sizing: border-box;
    box-sizing: border-box;
  }
  html {
    -webkit-tap-highlight-color: rgba(0, 0, 0, 0);
  }
  body {
    font-family: "Helvetica Neue", Helvetica, Arial, sans-serif;
    line-height: 1.66666667;
    font-size: 13px;
    color: #333333;
    background-color: #ffffff;
    margin: 2em 1em;
  }
  p {
    margin: 0 0 10px;
    font-size: 13px;
  }
  .alert.alert-info {
    padding: 15px;
    margin-bottom: 20px;
    border: 1px solid transparent;
    background-color: #f5f5f5;
    border-color: #8b8d8f;
    color: #363636;
    margin-top: 30px;
  }
  .alert p {
    padding-left: 35px;
  }
  a {
    color: #0088ce;
  }

  ul {
    position: relative;
    padding-left: 51px;
  }
  p.info {
    position: relative;
    font-size: 15px;
    margin-bottom: 10px;
  }
  p.info:before, p.info:after {
    content: "";
    position: absolute;
    top: 9%;
    left: 0;
  }
  p.info:before {
    content: "i";
    left: 3px;
    width: 20px;
    height: 20px;
    font-family: serif;
    font-size: 15px;
    font-weight: bold;
    line-height: 21px;
    text-align: center;
    color: #fff;
    background: #4d5258;
    border-radius: 16px;
  }

  @media (min-width: 768px) {
    body {
      margin: 4em 3em;
    }
    h1 {
      font-size: 2.15em;}
  }

  </style>
  </head>
  <body>
    <div>
      <h1>Application is not available</h1>
      <p>The application is currently not serving requests at this endpoint. It may not have been started or is still starting.</p>

      <div class="alert alert-info">
        <p class="info">
          Possible reasons you are seeing this page:
        </p>
        <ul>
          <li>
            <strong>The host doesn't exist.</strong>
            Make sure the hostname was typed correctly and that a route matching this hostname exists.
          </li>
          <li>
            <strong>The host exists, but doesn't have a matching path.</strong>
            Check if the URL path was typed correctly and that the route was created using the desired path.
          </li>
          <li>
            <strong>Route and path matches, but all pods are down.</strong>
            Make sure that the resources exposed by this route (pods, services, deployment configs, etc) have at least one pod running.
          </li>
        </ul>
      </div>
    </div>
  </body>
</html>
```

while it works with httpie client

```
http --verify=no -f https://secure-sso-sso.ose.spring-boot.osepool.centralci.eng.rdu2.redhat.com/auth/realms/master/protocol/openid-connect/token \
username=admin password=admin \
client_secret=1daa57a2-b60e-468b-a3ac-25bd2dc2eadc client_id=demoapp \
grant_type=password
HTTP/1.1 200 OK
Connection: keep-alive
Content-Length: 4344
Content-Type: application/json
Date: Wed, 08 Nov 2017 18:19:40 GMT
Server: JBoss-EAP/7
X-Powered-By: Undertow/1

{
    "access_token": "eyJhbGciOiJSUzI1NiIsInR5cCIgOiAiSldUIiwia2lkIiA6ICJRek1nbXhZMUhrQnpxTnR0SnkwMm5jNTNtMGNiWDQxV1hNSTU1MFo4MGVBIn0.eyJqdGkiOiIzYTU2ZjAyYS0xZWFiLTQ5MmItYjEzYi02ODQ2YzhlZDE1YWQiLCJleHAiOjE1MTAxNjU3ODAsIm5iZiI6MCwiaWF0IjoxNTEwMTY1MTgwLCJpc3MiOiJodHRwczovL3NlY3VyZS1zc28tc3NvLm9zZS5zcHJpbmctYm9vdC5vc2Vwb29sLmNlbnRyYWxjaS5lbmcucmR1Mi5yZWRoYXQuY29tL2F1dGgvcmVhbG1zL21hc3RlciIsImF1ZCI6ImRlbW9hcHAiLCJzdWIiOiIyNjI2NzdmYS0xMDJiLTRkNjAtOWE2MS1iMWNhMmFmZDY2M2EiLCJ0eXAiOiJCZWFyZXIiLCJhenAiOiJkZW1vYXBwIiwiYXV0aF90aW1lIjowLCJzZXNzaW9uX3N0YXRlIjoiNmE2YzM4ZDgtOGJkZi00MmVjLTg3MTktZWUxMGUxN2M3Mzk4IiwiYWNyIjoiMSIsImNsaWVudF9zZXNzaW9uIjoiYjA1YWIyMTYtYzI2My00MmQ4LTkxMDUtNWU3ZjU5ZDJjNWIxIiwiYWxsb3dlZC1vcmlnaW5zIjpbIioiXSwicmVhbG1fYWNjZXNzIjp7InJvbGVzIjpbImNyZWF0ZS1yZWFsbSIsImFkbWluIl19LCJyZXNvdXJjZV9hY2Nlc3MiOnsibWFzdGVyLXJlYWxtIjp7InJvbGVzIjpbInZpZXctaWRlbnRpdHktcHJvdmlkZXJzIiwidmlldy1yZWFsbSIsIm1hbmFnZS1pZGVudGl0eS1wcm92aWRlcnMiLCJpbXBlcnNvbmF0aW9uIiwiY3JlYXRlLWNsaWVudCIsIm1hbmFnZS11c2VycyIsInZpZXctYXV0aG9yaXphdGlvbiIsIm1hbmFnZS1ldmVudHMiLCJtYW5hZ2UtcmVhbG0iLCJ2aWV3LWV2ZW50cyIsInZpZXctdXNlcnMiLCJ2aWV3LWNsaWVudHMiLCJtYW5hZ2UtYXV0aG9yaXphdGlvbiIsIm1hbmFnZS1jbGllbnRzIl19LCJhY2NvdW50Ijp7InJvbGVzIjpbIm1hbmFnZS1hY2NvdW50Iiwidmlldy1wcm9maWxlIl19fSwibmFtZSI6IiIsInByZWZlcnJlZF91c2VybmFtZSI6ImFkbWluIn0.Mq0-IU7xloj5kLGk7rWNIKs1y_vkK-3bKb0hOmVPxIsJPOD7WbnMKATNxWXbrC1acVToLj1DK53bnAYcUTNCgORS-xxn-9B_IMTokm3cbhMmh0OR6ORUSCk3FEODjhOu0MIIzZdt-ZRceo19dCPTiEqyEBcmzlWJ6j36lMAXNglbG2_Vmru5ARbN_QZrnKVMnM8zT0woXugH45KUjteOWWK8K_zchgZowiPX9aRsvZLoOupRYpfuY9eG3uMNsbYVhC4OPvn4sqNCRu5O-fBLrmhNkzbUSv60-HePdju3LUL471GdO1cdCrjrya5DFvWhGds0aWxsRciCOGrZSdgh7g",
```

### PR DESCRIPTION
**Pull request by rhoads-zach**

### CODE REVIEW

File: docs/topics/proc_interacting-with-the-secured-booster-on-the-command-line.adoc [hunk 1]
```diff
@@ -41,6 +41,8 @@ $ curl -sk -X POST https://<SSO_AUTH_SERVER_URL>/auth/realms/$REALM/protocol/ope
 ----
 +
 NOTE: The `-sk` option tells curl to ignore failures resulting from self-signed certificates. Do not use this option in a production environment.
++
+IMPORTANT: On macOS, you must have `curl` version `7.56.1` or greater installed.
```
Comment (tradej) [inline docs/topics/proc_interacting-with-the-secured-booster-on-the-command-line.adoc]: I would put the information into the note above (the MInimalism initiative commented that our docs have too many admonitions, especially close together, which makes reading hard). Also, consider adding the information @metacosm noted about building curl with OpenSSL.

**Comment by rhoads-zach** [comment]
thanks @tradej and @metacosm. fixed

**Comment by rhoads-zach** [comment]
merging.

**Comment by rhoads-zach** [comment]
merged: a4035c5106e2c58d461048aef865975334a29483

**Pull request closed without merging**